packages/map-template/src/components/Search/Search.scss

```diff
@@ -18,14 +18,16 @@
         flex: 1 1 auto;
         margin-top: var(--spacing-x-small);
 
-        > * {
-            height: fit-content; /* fixing issue on mobile Safari causing them to bee too large in case of just one search result */
+        >* {
+            height: fit-content;
+            /* fixing issue on mobile Safari causing them to bee too large in case of just one search result */
         }
     }
 
     &__scroll-buttons {
         position: fixed;
-        left: calc(50% + 284px + 20px); /* Half viewport width + half modal width + gap */
+        left: calc(50% + 284px + 20px);
+        /* Half viewport width + half modal width + gap */
```

Comment:
It's confusing that the comment is now below what it is explaining. The same for the `height: fit-content` above.